webapp/channels/src/components/invitation_modal/invite_as.tsx

```diff
@@ -83,7 +81,7 @@ export default function InviteAs(props: Props) {
         if (isFreeTrial) {
             ctaExtraContentMsg = formatMessage({id: 'free.professional_feature.professional', defaultMessage: 'Professional feature'});
         } else {
-            ctaExtraContentMsg = (hasPriorTrial || cloudFreeDeprecated) ? formatMessage({id: 'free.professional_feature.upgrade', defaultMessage: 'Upgrade'}) : formatMessage({id: 'free.professional_feature.try_free', defaultMessage: 'Professional feature- try it out free'});
+            ctaExtraContentMsg = (hasPriorTrial) ? formatMessage({id: 'free.professional_feature.upgrade', defaultMessage: 'Upgrade'}) : formatMessage({id: 'free.professional_feature.try_free', defaultMessage: 'Professional feature- try it out free'});
```

Comment:
Is this correct? The rest of these behave like `cloudFreeDeprecated` is always true but if that's the case, but this one looks like how it would behave if it was false

Reply:
The `cloudFreeDeprecated` was necessary here to show the "Upgrade" button to those workspaces who were on Cloud Free leading up to the deprecation of that SKU in July 2023. As of July 2023 it's impossible to have a Cloud workspace, that isn't on a trial, and also not a paying subscription. This statement is here to show the "upgrade" button for _self hosted_ workspaces in the event they've already had a trial, or to show them the "try it free" button if they've not yet trialed